crawler/dockercontainer.py

```diff
@@ -344,33 +344,37 @@ def _get_logfiles_list(self,
         for logdict in self.log_file_list:
             name = logdict['name']
             _type = logdict['type']
-            log_source = rootfs_path + name
-            log_dest = host_log_dir + name
-            if "*" in log_source:
-               source_unglob_list = glob.glob(log_source)
-            else:
-               source_unglob_list = [ log_source ]
-
-            logger.debug('GLOB LIST %s' % source_unglob_list) 
-            for source_unglob in source_unglob_list:
-               logger.debug('SOURCE GLOB %s' % source_unglob) 
-               if rootfs_path in source_unglob:
-                   dest_unglob = host_log_dir + source_unglob.split(rootfs_path, 1)[1]
-               else:
-                   dest_unglob = host_log_dir + source_unglob
-
-               logger.debug('DEST GLOB %s' % dest_unglob) 
 
+            # assuming mount source or destination does not contain '*'
+            for mount in self.inspect['Mounts']:
```

Comment:
you tested in docker 1.11 right? creating a new issue to add travis tests for something before 1.10. Relying on inspect naming conventions gave us some pain in the past. Next version they could change it to "mounts" or "The_mounts" or whatever.